AY2223S1-CS2103-W14-1/tp
# Implement Undo Command

Closes https://github.com/AY2223S1-CS2103-W14-1/tp/issues/98

This PR introduces the 'undo' command.

Undo reverses any previous commands executed by the user. Can be executed multiple times to undo successive commands.

To test:
- `add -p n/test a/test`
- `add -c n/test a/test`
- `undo`
- `undo`
- `undo`
- You should get a message saying "no more commands to undo"

Note: `undo` will also undo commands that have no effect, such as `help`. In that case, nothing happens. Is this intended behavior? Or should I only undo commands that change the property directory.
